components/pedigree/resources/src/main/resources/pedigree/externalEndpoints.js

```diff
@@ -107,6 +107,10 @@ define([
             return this.vocabulariesService + "/omim";
         },
 
+        getPatientDeleteURL: function(patientId) {
+            return XWiki.contextPath + "/rest/patients/" + patientId + "?method=DELETE";
```

Comment:
I wonder if we need to add that `rand` parameter for IE. TODO: check if deleting the same patient twice works or not in IE.

Reply:
I wonder if those `rand`s are there for IE8 or IE9 (pedigree can be used in read-only mode in IE8, not sure if we still care in 2017/2018)